apps/angular-example/src/app/app-routing.module.ts

```diff
@@ -19,9 +19,9 @@ import { forkJoin } from 'rxjs';
 })
 export class AppRoutingModule {
   constructor(private abby: Abby, private router: Router) {
-    forkJoin({
-      angularTest: abby.getVariant('AngularTest'),
-      angularFlag: abby.getFeatureFlagValue('AngularFlag'),
+    combineLatest({
```

Comment:
i dont know if we want to use combineLatest here? forkJoin would emit once and complete. combineLatest does not. do we really want to change our router-config possibly over and over again?

Reply:
I would say we want `combineLatest` here.
`forkJoin` is described as:
> Wait for Observables to complete and then combine last values they emitted; complete immediately if an empty array is passed.
in the docs: https://rxjs.dev/api/index/function/forkJoin

The issue is, that our Observables never complete, as they can be changed by the devtools during runtime.
In a live-application, we could use `forkJoin` here, because the values do not change and thus the Observables complete. But in the demo it won't update.

We could add an entry to the docs, that explain this.

Reply:
there should be definitely a comment, even at the code. its an example, it should teach users how to use it! :)) this code will be copy-pasted.